base-images/virtualization_images.yml

```diff
@@ -6,6 +6,7 @@ BASE_DEBIAN_BOOKWORM_SLIM: "debian:bookworm-slim@sha256:e9ac68ffde903b241342267a
 BASE_CONTAINER_REGISTRY: "registry:2.8.3@sha256:ac0192b549007e22998eb74e8d8488dcfe70f1489520c3b144a6047ac5efbe90"
 BASE_GOLANG_22_BOOKWORM: "golang:1.22.8-bookworm@sha256:9e7db50b9858e9cd804043200f1e6acd5a11111151ce886951c9fe3523002cea"
 BASE_GOLANG_23_BOOKWORM: "golang:1.23.6-bookworm@sha256:441f59f8a2104b99320e1f5aaf59a81baabbc36c81f4e792d5715ef09dd29355"
+BASE_GOLANG_24_BOOKWORM: "golang:1.24.2-bookworm@sha256:79390b5e5af9ee6e7b1173ee3eac7fadf6751a545297672916b59bfa0ecf6f71"
```

Comment:
There is a Go 1.24 in base image from deckhouse:

```
builder/golang-bookworm-1.24: "sha256:2029e67a074fc2469d5331a273c2b37300a6839ab94df47e1718d576430b3dd7" # from: golang:1.24.2-bookworm

```

https://github.com/deckhouse/virtualization/blob/main/base-images/deckhouse_images.yml#L18

@universal-itengineer what do you think?